platform/ios/app/MBXOfflinePacksTableViewController.m

```diff
@@ -102,7 +102,9 @@ - (IBAction)addCurrentRegion:(id)sender {
             name = nameField.placeholder;
         }
 
+        NSString *fontFamilyName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLIdeographicFontFamilyName"];
         MGLTilePyramidOfflineRegion *region = [[MGLTilePyramidOfflineRegion alloc] initWithStyleURL:mapView.styleURL bounds:mapView.visibleCoordinateBounds fromZoomLevel:mapView.zoomLevel toZoomLevel:mapView.maximumZoomLevel];
+        region.includesIdeographicGlyphs =fontFamilyName ? YES : NO;
```

Comment:
FYI, it’s fine to just say `fontFamilyName` here; Objective-C type checking is only as strict as C, not like Swift, so you don’t even have to say `!= nil`.